src/pages/Earn/GmxAndVotingPowerCard.tsx

```diff
@@ -0,0 +1,353 @@
+              position="bottom-end"
+              renderContent={() => {
+                return (
+                  <>
+                    <StatsTooltipRow
+                      label={t`GMX`}
+                      value={`${formatKeyAmount(
+                        processedData,
+                        "extendedGmxTrackerRewards",
+                        18,
+                        4
+                      )} ($${formatKeyAmount(processedData, "extendedGmxTrackerRewardsUsd", USD_DECIMALS, 2, true)})`}
+                      showDollar={false}
+                    />
+                    <StatsTooltipRow
+                      label="Escrowed GMX"
+                      value={`${formatKeyAmount(processedData, "stakedGmxTrackerRewards", 18, 4)} ($${formatKeyAmount(
+                        processedData,
+                        "stakedGmxTrackerRewardsUsd",
+                        USD_DECIMALS,
+                        2,
+                        true
+                      )})`}
+                      showDollar={false}
+                    />
+                    {isAnyFeeGmxTrackerRewards && (
+                      <StatsTooltipRow
+                        label={`${nativeTokenSymbol} (${wrappedTokenSymbol})`}
+                        value={`${formatKeyAmount(
+                          processedData,
+                          "feeGmxTrackerRewards",
+                          18,
+                          4
+                        )} ($${formatKeyAmount(processedData, "feeGmxTrackerRewardsUsd", USD_DECIMALS, 2, true)})`}
+                        showDollar={false}
+                      />
+                    )}
+                  </>
+                );
+              }}
+            />
+          </div>
+        </div>
+        <div className="App-card-divider"></div>
+        <div className="App-card-row">
+          <div className="label">
+            <Trans>Total Staked</Trans>
+          </div>
+          <div>
+            {totalGmxStaked === undefined && "..."}
+            {(totalGmxStaked !== undefined && (
+              <Tooltip
+                position="bottom-end"
+                className="whitespace-nowrap"
+                handle={
+                  formatAmount(totalGmxStaked, 18, 0, true) +
+                  " GMX" +
+                  ` ($${formatAmount(stakedGmxSupplyUsd, USD_DECIMALS, 0, true)})`
+                }
+                renderContent={() => (
+                  <ChainsStatsTooltipRow
+                    showDollar={false}
+                    decimalsForConversion={18}
+                    symbol="GMX"
+                    entries={stakedEntries}
+                  />
+                )}
+              />
+            )) ||
+              null}
+          </div>
+        </div>
+        <div className="App-card-row">
+          <div className="label">
+            <Trans>Total Supply</Trans>
+          </div>
+          {totalGmxSupply === undefined && "..."}
```

Comment:
This can be a single ternary instead of two mixed conditions

Reply:
done